Jenkinsfile_k8s

```diff
@@ -0,0 +1,5 @@
+buildDockerAndPublishImage('wiki', [
+    mainBranch: 'main',
+    automaticSemanticVersioning: true,
+    gitCredentials: 'github-app-infra'
+])
```

Comment:
```suggestion
])

```

Reply:
Nitpick: As per POSIX, all lines should end with an "endline" character (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline -> https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)